package.json

```diff
@@ -1,19 +1,30 @@
 {
-  "name": "mythicalselfbot",
+  "name": "mythical-selfbot",
   "version": "1.0.0",
   "description": "A Mythical Selfbot for Discord",
   "main": "bot.js",
   "scripts": {
-    "test": "node bot.js",
-    "start": "node bot.js"
+    "nm": "nodemon --harmony bot.js",
+    "test": "node --harmony bot.js",
+    "start": "node --harmony bot.js"
   },
-  "author": "Thunder33345, forked off HimbeersaftLP",
-  "license": "dunno ¯\\_(ツ)_/¯",
+  "author": "Thunder33345",
+  "fork": "HimbeersaftLP",
+  "contributors": {
```

Comment:
i doubt this will be feasible in the future
i dont fancy having an endless list of contributors